docs/content/build-image.md

```diff
@@ -12,70 +12,75 @@ When you create a new bundle with Porter, your project is bootstrapped with a sa
 To create a new CNAB with Porter, you first run `porter create`. The generated `porter.yaml` will look like this:
 
 ```yaml
-# This is the configuration for Porter
-# You must define steps for each action, but the rest is optional
-# See https://porter.sh/author-bundles for documentation on how to configure your bundle
-# Uncomment out the sections below to take full advantage of what Porter can do!
 
 name: HELLO
 version: 0.1.0
 description: "An example Porter configuration"
-# TODO: update the registry to your own, e.g. myregistry/porter-hello:v0.1.0
 tag: getporter/porter-hello:v0.1.0
 
-# Uncomment the line below to use a template Dockerfile for your invocation image
-#dockerfile: Dockerfile.tmpl
-
 mixins:
   - exec
 
 install:
   - exec:
       description: "Install Hello World"
-      command: bash
-      flags:
-        c: echo Hello World
+      command: ./helpers.sh
+      arguments:
+        - install
 
 upgrade:
   - exec:
       description: "World 2.0"
-      command: bash
-      flags:
-        c: echo World 2.0
+      command: ./helpers.sh
+      arguments:
+        - upgrade
 
 uninstall:
   - exec:
       description: "Uninstall Hello World"
-      command: bash
-      flags:
-        c: echo Goodbye World
-
-
-# See https://porter.sh/author-bundles/#dependencies
-#dependencies:
-#  mysql:
-#    tag: getporter/mysql:v0.1.2
-#    parameters:
-#      database-name: wordpress
-
-# See https://porter.sh/wiring/#credentials
-#credentials:
-#  - name: kubeconfig
-#    path: /root/.kube/config
-
+      command: ./helpers.sh
+      arguments:
+        - uninstall
 ```
 
 After the scaffolding is created, you may edit the _porter.yaml_ and modify the `tag: getporter/porter-hello:v0.1.0` element representing the bundle tag to include a Docker registry that you can push to. Note that the bundle is not pushed during the `porter build` workflow.
 
-Once you have modified the `porter.yaml`, you can run `porter build` to generate your first invocation image.  Here we add the `--verbose` flag to see all of the output:
+Once you have modified the `porter.yaml`, you can run `porter build` to generate your first invocation image.  Here we add the `--debug` flag to see all of the output:
 
 ```console
-$ porter build --verbose
+$  porter build --debug
+Resolved porter binary from /usr/local/bin/porter to /Users/sigje/.porter/porter
```

Comment:
I think leaving the home directory in is fine. We've left it for other example outputs and people seem to understand that so far (or at least no one has commented).